.snyk

```diff
@@ -1,10 +1,17 @@
-version: v1.38.1
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.7.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
-  'SNYK-RUBY-NOKOGIRI-20299':
+  SNYK-RUBY-NOKOGIRI-20299:
     - '* > nokogiri':
-        reason: 'We do not opt in to DTDLOAD and do not opt out of NONET'
+        reason: We do not opt in to DTDLOAD and do not opt out of NONET
         expires: '2018-01-01T00:00:00.000Z'
-  'SNYK-RUBY-ACTIONCABLE-20338':
+  SNYK-RUBY-ACTIONCABLE-20338:
     - '* > actioncable':
-        reason: 'We do not use actioncable'
+        reason: We do not use actioncable
         expires: '2017-09-10T00:00:00.000Z'
+# patches apply the minimum changes required to fix a vulnerability
+patch:
+  'npm:debug:20170905':
+    - app > mongoose > mquery > debug:
+        patched: '2017-09-26T23:05:04.504Z'
```

Comment:
I'm not sure what this is doing. @jmhooper Any idea?

Reply:
Yes, this is Snyk's patch feature. You can read about how the different ways to mitigate vulnerabilities in their docs [here](https://snyk.io/docs/using-snyk/#wizard).

The gist is that the Snyk wizard was not able to find a direct upgrade path that fixed the issue without breaking changes. So essentially no fix by doing `debug@1.0.x` to `debug@1.0.y`. What they do instead is have "patch" which is a change they'll actually make to the node module when you run `snyk protect`. The :-1: here is that you have to actually run `npm run snyk-protect` after installing the node module in order for it to do anything.

We should look at whether we are actually vulnerable to anything introduced by this `debug` module, and if we are, prefer to upgrade the module and fix anything that breaks as a result.